app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
```

Comment:
well, if you use the id ``login-users-help`` outside the login page, it would look weird regarding the naming convention for ids...